### PR TITLE
Added unexpected error handling module, modified 

### PR DESCRIPTION
**Pull request by AbhiramKrishnaM**
Additional:
1) Added nodemon package.
2) Added "dev" to script for file change watching in development

### CODE REVIEW

File: server/index.js
Comment (lugenx) [inline server/index.js:12]: - should we rename the function as just `errorHandler`, I think that would be simpler.
- also can we make the filename same as the function name? instead of `catch.js`, let's make it `errorHandler.js`

File: server/index.js
Comment (lugenx) [inline server/index.js:41]: it's nice to write this part and check the handler, but let's leave this route out of the PR.

File: server/package.json
Comment (lugenx) [inline server/package.json:21]: i know this is handy tool, but can we wait to install it for now until it's decided? you can install nodemon globally on your computer for now if you want to use it.

**Review by lugenx — CHANGES_REQUESTED** [review]
good work @AbhiramKrishnaM; thank you for this, i have just a couple comments. I would also like to ask @mannan0819 's input here.

File: server/index.js
Comment (AbhiramKrishnaM) [reply server/index.js]: Yes, changed

File: server/index.js
Comment (AbhiramKrishnaM) [reply server/index.js:36]: Removed check route

File: server/package.json
Comment (AbhiramKrishnaM) [reply server/package.json]: removed nodemon

**Comment by AbhiramKrishnaM** [comment]
Updated as requested

**Comment by AbhiramKrishnaM** [comment]
Can you check this. I have updated the pages as requested

**Comment by lugenx** [comment]
hey! thanks for your contribution! Just a heads up - next time, wait for our final review before closing your pull request. i appreciate your interest in our project. cheers!

**Pull request closed without merging**
